boards/arm/mec1501modular_assy6885/mec1501modular_assy6885_defconfig

```diff
@@ -17,13 +17,5 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_PINMUX=y
 CONFIG_GPIO=y
-CONFIG_PS2=y
-CONFIG_ADC=y
-CONFIG_PWM=y
-CONFIG_SENSOR=y
-CONFIG_I2C=y
-CONFIG_I2C_INIT_PRIORITY=60
-CONFIG_ESPI=y
-CONFIG_ESPI_PERIPHERAL_UART_SOC_MAPPING=1
```

Comment:
are you sure you can remove `CONFIG_ESPI_PERIPHERAL_UART_SOC_MAPPING`?

Reply:
added to defconfig, thanks